sparkly/testing.py

```diff
@@ -120,28 +147,132 @@ def tearDown(self):
         for fixture in self.fixtures:
             fixture.teardown_data()
 
-    def assertDataFrameEqual(self, actual_df, expected_data, fields=None, ordered=False):
+    def assertDataFrameEqual(self,
+                             actual_df,
+                             expected_data,
+                             fields=None,
+                             ordered=False,
+                             places=None):
         """Ensure that DataFrame has the right data inside.
 
         Args:
             actual_df (pyspark.sql.DataFrame|list[pyspark.sql.Row]): Dataframe to test data in.
             expected_data (list[dict]): Expected dataframe rows defined as dicts.
             fields (list[str]): Compare only certain fields.
             ordered (bool): Does order of rows matter?
+            places (int|None): Compare floats up to given decimal places.
```

Comment:
I'd prefer to set this as allowed deviation (float), like abs(x'-x) < dev, this is more precise and common. The profit also that you can call the parameter with a greek letter 🇬🇷  😄  eps (epsilon) or delta